MapboxCoreNavigationTests/DistanceFormatterTests.swift

```diff
@@ -0,0 +1,55 @@
+import XCTest
+import CoreLocation
+@testable import MapboxCoreNavigation
+
+let oneMile: CLLocationDistance = metersPerMile
+
+class DistanceFormatterTests: XCTestCase {
+    
+    var spokenDistanceFormatter = SpokenDistanceFormatter(approximate: true)
+    var distanceFormatter = DistanceFormatter(approximate: true)
+    
+    override func setUp() {
+        super.setUp()
+        spokenDistanceFormatter.unitStyle = .long
+    }
+    
+    func assertDistance(_ distance: CLLocationDistance, spoken: String, displayed: String) {
+        let spokenString = spokenDistanceFormatter.string(from: distance)
+        let displayedString = distanceFormatter.string(from: distance)
+        XCTAssert(spokenString.contains(spoken), "Spoken '\(spokenString)' should be equal to '\(spoken)'")
+        XCTAssert(displayedString.contains(displayed), "Displayed: '\(displayedString)' should be equal to \(displayed)")
+    }
+    
+    func testDistanceFormatters_US() {
+        spokenDistanceFormatter.numberFormatter.locale = Locale(identifier: "en-US")
+        distanceFormatter.numberFormatter.locale = Locale(identifier: "en-US")
+        
+        assertDistance(oneMile/2,   spoken: "a half mile",        displayed: "0.5 mi")
```

Comment:
We might want to add tests for quarter and three quarter miles. It's not clear what the behavior should be from the current test cases.